application/helpers/database_helper.php

```diff
@@ -103,14 +103,3 @@ function dbSelectTablesLike($table)
             safeDie("Couldn't create 'select tables like' query for connection type '" . Yii::app()->db->getDriverName() . "'");
     }
 }
-
-/**
```

Comment:
This function is used in alot of places. Why did you move it to the import_helper? You will now have to check all places where it is used and makre sure that import_helper is loaded, instead.

Reply:
I was moving another function and by accident moved this one as well. Thanks for pointing it out.